### PR TITLE
Add new information about when torrent was added to transmission in the Information Tab in transmission-qt

### LINKED ISSUE #1729: [Feature request] Show when a torrent was added in "Torrent Properties" window
As of Transmission 2.94, the "Information" tab of the "Torrent Properties" window shows the following:

- Activity
  - Have: bytes/%
  - Availability: %
  - Uploaded: bytes
  - Downloaded: bytes
  - State
  - Running time: n days, m hours
  - Last activity
  - Error
- Details
  - Size
  - Location
  - Hash
  - Privacy
  - Origin
  - Comment

It would be nice, if the "Information" tab of the "Torrent Properties" window also showed _when_ a particular torrent was added to Transmission. Since there is "View > Sort by Age", I assume that Transmission stores some kind of information regarding when a torrent was added.

Thanks.

### PR DESCRIPTION
**Pull request by sewe2000**
Resolves  #1729 in qt version.

### CODE REVIEW

File: qt/DetailsDialog.ui [hunk 1]
```diff
@@ -371,6 +371,7 @@
            </property>
           </widget>
          </item>
+
```
Comment (ile6695) [inline qt/DetailsDialog.ui:374]:
```suggestion
```

**Review by ile6695 — COMMENTED** [review]
Builds as expected and looks good. There's one extra newline character in `qt/DetailsDialog.ui`

**Review by ile6695 — APPROVED** [review]
LGTM

**Review by ckerr — APPROVED** [review]
Thanks!

**Comment by rybak** [comment]
@sewe2000 thank you for working on this. Thanks to @ile6695 for review. And thank you, @ckerr, for maintaining Transmission.

**Pull request merged**